docs/guide/cli.md

```diff
@@ -55,6 +55,36 @@ export default {
 
 Run only [benchmark](https://vitest.dev/guide/features.html#benchmarking-experimental) tests, which compare performance results.
 
+### `vitest init`
+
+`vitest init <name>` can be used to setup project configuration. At the moment, it only supports [`browser`](/guide/browser) value:
+
+```bash
+vitest init browser
+```
+
+### `vitest list`
```

Comment:
This feature should be marked as breaking change.

Some time ago we received feedback that adding these "top level" commands can cause weird behaviour for end users. These will always mix with the CLI's test file filter. 

Earlier if they ran `vitest list` and expected it to run `user-list.test.ts`, `product-list.test.ts`, now they will see this new command popping up.

Reply:
Or well, I don't know about the breaking change part. Not sure if we decided whether new commands should be marked as such.

Reply:
It is a breaking change, yes